### PR TITLE
Manage clipboard using pbcopy/pbpaste commands.

### PR DESCRIPTION
**Pull request by mecid**
pbcopy and pbpaste implemented.

### CODE REVIEW

File: OpenTerm.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -560,12 +564,12 @@
 				CODE_SIGN_ENTITLEMENTS = OpenTerm/OpenTerm.entitlements;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 6G5LMQ72D8;
+				DEVELOPMENT_TEAM = 2952RH3AR7;
```
Comment (IMcD23) [inline OpenTerm.xcodeproj/project.pbxproj]: Please don't check in changes to development team or bundle identifier.

Comment (mecid) [reply OpenTerm.xcodeproj/project.pbxproj]: Got it. Thanks.

Comment (mecid) [reply OpenTerm.xcodeproj/project.pbxproj]: Fixed.

**Comment by mecid** [comment]
Commands renamed to pbcopy and pbpaste, development team also restored to original one.

**Comment by louisdh** [comment]
Nice, very useful commands!

**Pull request merged**